### DISCUSSION TITLE
New/Duplicate/Edit Bookmark is disabled

### DISCUSSION THREAD

**Discussion by MSG7573** [iterate-ch/cyberduck#16409] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**
Now i am not able to add more bookmarks, i have 31 total bookmarks.

**To Reproduce**
<img width="213" alt="Screenshot 2024-09-26 at 11 15 37" src="https://github.com/user-attachments/assets/0fc6a4ca-5321-422e-beb8-c438a39bc5bd">

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [macOS 15.0 (24A335)]
 - Version [9.0.2(42108)]

**Comment by dkocher** [comment]
I suppose you are viewing the _History_ or _Bonjour_. Choose _Bookmarks → Toggle Bookmarks (⌘B)_.
